include/wx/msw/textctrl.h

```diff
@@ -132,6 +132,7 @@ class WXDLLIMPEXP_CORE wxTextCtrl : public wxTextCtrlBase
     virtual bool SetBackgroundColour(const wxColour& colour) wxOVERRIDE;
     virtual bool SetForegroundColour(const wxColour& colour) wxOVERRIDE;
     virtual bool SetFont(const wxFont& font) wxOVERRIDE;
+    wxFont GetFont() const;
```

Comment:
Sorry, this doesn't seem like a good idea: either we need to make `GetFont()` virtual in the base class or rename this to something like `MSWGetScaledFont()` to avoid hiding the base class method.

Ideal would probably be to update `m_font`, so that `GetFont()` returns the font of the same size as the one actually used by the control, but keep using unscaled `HFONT` for the native control.

Reply:
I'll revert it to the original fix (using `WXAdjustToPPI` in `DoGetSizeFromTextSize`).